internal/resourcepaths/silo.go

```diff
@@ -0,0 +1,10 @@
+package resourcepaths
+
+const (
+	// silo container resources paths
+	SiloDeviceResourcePath          string = "Container/Devices/Generic"    //nolint
+	SiloMappedDirectoryResourcePath string = "Container/MappedDirectories"  //nolint
+	SiloMappedPipeResourcePath      string = "Container/MappedPipes"        //nolint
+	SiloMemoryResourcePath          string = "Container/Memory/SizeInMB"    //nolint
+	SiloRegistryFlushStatePath      string = "Container/RegistryFlushState" //nolint
+)
```

Comment:
should we use the same linter tags here as for the vm resource paths?

Reply:
Which is preferred

Reply:
If both work I'd rather just annotate the whole const block like the vm ones

Reply:
Done